src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -71,6 +74,16 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
         return MONGO_CONNECTION_EXPECTED;
     }
 
+    public static validatePostgresConnectionString(value: string): string | undefined {
+        value = value ? value.trim() : '';
+
+        if (value && value.match(/^postgres:\/\//)) {
+            return undefined;
+        }
+
+        return POSTGRES_CONNECTION_EXPECTED;
```

Comment:
You should only make a constant for something like `POSTGRES_CONNECTION_EXPECTED` if you're going to use it in more than one place